upbound_uxp/manifest.json

```
@@ -0,0 +1,53 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "f1427ca4-4b88-484a-a7c5-9cc66542b0d9",
+  "app_id": "upbound-uxp",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "",
+    "title": "Upbound UXP",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
```

```suggestion
      "Supported OS::macOS",
      "Offering::Integration"
```
Please also add any relevant categories here, as well as the types of data that this integration queries and submits. For more details please see this [documentation](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags).

---

Added categories.